Subsurface/Node/Common/Peripherals/inc/PWMInterface.h

```diff
@@ -0,0 +1,45 @@
+#include "stm32f3xx_hal.h"
+#include <cstdint>
+
+#ifndef PWMINTERFACE_H
+#define PWMINTERFACE_H
+
+class PWMInterface
+{
+public:
+    // Constructor
```

Comment:
Can we also have a constructor that just takes a `TIM_HandleTypeDef*` so that we also have the ability to just use the PWM the way it was configured in CubeMX?